.github/workflows/update-dependencies.yml

```diff
@@ -12,7 +12,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - uses: excitedleigh/setup-nox@v2.0.0
```

Comment:
FYI, the latest commit of `excitedleigh/setup-nox`, it's just that @excitedleigh has not made a release.

IMO using the nox action is fine, though.

Reply:
> FYI, the latest commit of `excitedleigh/setup-nox`, it's just that @excitedleigh has not made a release.

I've been using the latest commit of `excitedleigh/setup-nox` in the manylinux repo which was available before your PR was merged in `wntrblm/nox`

> IMO using the nox action is fine, though.

Given `wntrblm/nox` is now available as an action, maybe going with this is safer in the long run ? (given it's a composite action based on `setup-python@v3` it might be less likely to break)